jujugui/static/gui/src/app/app.js

```diff
@@ -1902,6 +1906,56 @@ YUI.add('juju-gui', function(Y) {
     },
 
     /**
+      Display post deployment help.
+
+      @param {String} entityId The entity ID of the charm or bundle.
+    */
+    _displayCanvasInfo: function(entityId) {
+      if(!window.juju_config.flags.canvasInfo){
+        return;
+      }
+      this._clearCanvasInfo();
+      const charmstore = this.get('charmstore');
+
+      const showEntityDetails = (id) => {
+        let url;
+        try {
+          url = window.jujulib.URL.fromString(id);
+        } catch (_) {
+          url = window.jujulib.URL.fromLegacyString(id);
+        }
+
+        const storeState = {
+          profile: null,
+          search: null,
+          store: url.path()
+        };
+
+        this.state.changeState(storeState);
+      };
+
+      this.get('charmstore').getEntity(entityId,
+        (err, entityData) => {
+          if (err) {
```

Comment:
Kind of disagree. This feature isn't a required part of the deployment and management of charms or bundles and we should avoid causing alarm and the feeling of something being broken.